scanomatic/data_processing/phenotyper.py

```diff
@@ -1745,7 +1745,7 @@ def _arr_tester(arr):
                 else:
                     raise ValueError("Any check failed")
             except ValueError:
-                return any(e.any() for e in arr.ravel())
+                return arr.ravel().any()
```

Comment:
have you validated that this change works with example in https://github.com/numpy/numpy/issues/10328

Reply:
it did crash it was for me, I updated this and checked that it handles the issue.